File: Semester 4/Programming and Development Methods/MPPProject/src/main/java/Domain/Credentials.java
package Domain;



public class Credentials implements IdEntity<Integer> {

    private int userId;
    private String password;

    public Credentials(int userId, String password) {
        this.userId = userId;
        this.password = password;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    @Override
    public Integer getId() {
        return userId;
    }

    @Override
    public void setId(Integer integer) {
        this.userId = integer;
    }
}

File: Semester 4/Programming and Development Methods/CSProject/Project/Project/Domain/Credentials.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Project.Domain
{
    class Credentials : IdEntity<int>
    {
        public Credentials(int userId, string password)
        {
            this.userId = userId;
            this.password = password;
        }

        public int userId { get; set; }
        public String password { get; set; }


        public int GetId()
        {
            return userId;
        }

        public void SetId(int id)
        {
            this.userId = id;
        }
    }
}
